src/sentry/static/sentry/app/views/settings/incidentRules/ruleConditionsForm.tsx

```diff
@@ -4,51 +4,55 @@ import styled from '@emotion/styled';
 import {addErrorMessage} from 'app/actionCreators/indicator';
```

Comment:
looks like a lot of changes happened here, but the changes are as follows:

1. ruleConditionsForm.tsx (the original form) is deleted
2. ruleConditionsFormWithGuiFilters.tsx --renamed--> ruleConditionsForm.tsx

So the new `ruleConditionsForm.tsx` is just the old `ruleConditionsFormWithGuiFilters.tsx`